Error in performance profile daily

An assertion error seems like it might be causing some issues. https://grpc-testing.appspot.com/view/Performance/job/gRPC_performance_profile_daily/16/consoleText
```
2017-04-17 11:44:09,578 START: /usr/bin/pypy
2017-04-17 11:44:09,767 WAITING: 0 queued, 8 jobs running, 0 complete, 0 failed
2017-04-17 11:44:09,835 Traceback (most recent call last):
  File "tools/profiling/latency_profile/profile_analyzer.py", line 179, in <module>
    if cs.add(inf):
  File "tools/profiling/latency_profile/profile_analyzer.py", line 131, in add
    self.stk.pop().finish(line)
  File "tools/profiling/latency_profile/profile_analyzer.py", line 87, in finish
    line['t']))
AssertionError: expected incoming_byte_stream_next, got grpc_exec_ctx_flush; thread=0; t0=10.006781 t1=10.006785
```
and a slightly different one:
```
2017-04-17 10:57:10,510 FAILED: /usr/bin/pypy [ret=1, pid=18067]
2017-04-17 10:57:10,510 WAITING: ETA 0.2 sec; 0 queued, 3 jobs running, 5 complete, 5 failed
2017-04-17 10:57:10,549 Traceback (most recent call last):
  File "tools/profiling/latency_profile/profile_analyzer.py", line 179, in <module>
    if cs.add(inf):
  File "tools/profiling/latency_profile/profile_analyzer.py", line 131, in add
    self.stk.pop().finish(line)
  File "tools/profiling/latency_profile/profile_analyzer.py", line 87, in finish
    line['t']))
AssertionError: expected incoming_byte_stream_next, got grpc_closure_run; thread=0; t0=10.653874 t1=10.653874
```

--- a/src/core/ext/transport/chttp2/transport/chttp2_transport.c
+++ b/src/core/ext/transport/chttp2/transport/chttp2_transport.c
@@ -2602,6 +2602,7 @@ static bool incoming_byte_stream_next(grpc_exec_ctx *exec_ctx,
       (grpc_chttp2_incoming_byte_stream *)byte_stream;
   grpc_chttp2_stream *s = bs->stream;
   if (s->unprocessed_incoming_frames_buffer.length > 0) {
+    GPR_TIMER_END("incoming_byte_stream_next", 0);
     return true;
   } else {
     gpr_ref(&bs->refs);

